modules/ROOT/pages/getting-started.adoc

```diff
@@ -20,11 +20,11 @@ The SDK provides a Maven plugin and a Maven archetype to create new projects. Th
 
 Install and verify that the following tools are working in your local environment:
 
-* Java Development Kit 8 (JDK 8) to compile and build your Java code.
+* Java Development Kit 8 (JDK 8) to compile and build your custom Java code.
 +
-NOTE: The maximum supported version for Java SDK is JDK 17, however, you can use JDK 17 only for running your application. Compilation with Java SDK must be done with JDK 8.
+NOTE: The maximum supported version for running your applications is JDK 17. However, if there is custom Java code in your application, then during packaging, the custom code is always compiled using JDK 8. 
```

Comment:
`the custom code is always compiled using JDK 8` -> `custom code MUST be compiled for Java language level 1.8`